superset-frontend/src/explore/components/controls/FixedOrMetricControl/index.jsx

```diff
@@ -127,7 +127,7 @@ export default class FixedOrMetricControl extends React.Component {
           <Collapse.Panel
             showArrow={false}
             header={
-              <Label onClick={() => undefined}>
```

Comment:
Can we git blame this and see why it was done in the first place?

Reply:
This was introduced as part of a migration from Bootstrap to AntD components in this [PR](https://github.com/apache/superset/pull/12920/files#diff-a7be32a17ea0cae5c0409a1c8ef7abd669132a56052f97a24aac78570f074cef). However, there doesn't seem to be additional context as to why this `onClick` is returning `undefined`.

Reply:
@michael-s-molina we are keeping this here unless you give us the green light to remove it